### ISSUE TITLE
Ventura

### ISSUE THREAD

**Issue by duyjack** [alucardness/macos_remote_management#1]
Have you tested it with ventura latest on Mac ( Apple silicon)?

**Comment by alucardness** [comment]
Yep, I've updated from Big Sur to the latest Ventura version 13.2.1

**Comment by duyjack** [comment]
Do you reinstall new MacOS ventura or you update to ventura from big sur ( don't reinstall new OS)?

**Comment by alucardness** [comment]
I was on Ventura, but did the factory reset for the first time and when I saw the Remote Management window I decided to try with latest version of Monterey, but still the problem persists there, so the only option was Big Sur. And then from Big Sur jumping straight to Monterey from the updates.

**Comment by X-DSI** [comment]
Hey there! I'm a newbie to Mac and macOS. I tried your commands but it says the sudo command is not recognized. I'm trying it from the terminal on the recovery mode. And also I cannot bypass the remote management screen even when the internet is not connected throughout the process. I'm on a MacBook Pro mid 2012, trying to install macOS Ventura using Opencore Legacy patcher.

**Comment by alucardness** [comment]
You must install Big Sur first, skip the wi-fi on setup screen, type commands after you reach desktop for first time and then updating to Ventura. I used to have a hackintosh with open core, but I'm not sure if the update will work just like that out of the box.

**Comment by drchandruj93** [comment]
Hi , thanks for posting .
This might be dumb question.

After this procedure, will this be  completely removed? I mean if we factory reset again, will the same remote management will pop-up?
I mean is there anyway to completely remove this remote management?
I m new to macOS. Bought a Mac from fb marketplace.now i m stuck on this.
